### DISCUSSION TITLE
Mountain Duck creates Network Mounts with User "NFS" for localized Folders

### DISCUSSION THREAD

**Discussion by mvosz** [iterate-ch/cyberduck#15994] [Q&A] [answered]
Why Mountain Duck create nfs shares? I cannot remember it was before an update. Is it normal or necessary? When not, how can I get rid of to minimize entries in my finders sidebar?

**Comment by dkocher** [comment, marked as answer]
The reason for the `.localized` suffix of the mount point that provides localization for the mount name is discussed in #15698.
